stepfunctions/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "common_tags" {}
+variable "name" {}
```

Comment:
Maybe "name" variable could be more descriptive?

Reply:
I've called it step_function_name